data/xml/W19.xml

```diff
@@ -7435,7 +7435,9 @@
       <editor><first>Lamia Hadrich</first><last>Belguith</last></editor>
       <editor><first>Fethi</first><last>Bougares</last></editor>
       <editor><first>Walid</first><last>Magdy</last></editor>
-      <editor><first>Imed</first><last>Zitouni</last></editor>
+      <editor><first>Imed</first><last>Zitouni</last></editor>	  	  
+	  <editor><first>Nadi</first><last>Tomeh</last></editor>
+	  <editor><first>Mahmoud</first><last>El-Haj</last></editor>
       <publisher>Association for Computational Linguistics</publisher>
```

Comment:
Please fix the indention to match its parents.

Reply:
just did